android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -387,20 +387,6 @@ public LiveData<Integer> getCoinTypeLd() {
             return mCoinTypeMutableLiveData;
         }
 
-        @Override
-        public String[] getEnabledKeyrings() {
-            ArrayList<String> keyRings = new ArrayList<>();
-            keyRings.add(BraveWalletConstants.DEFAULT_KEYRING_ID);
-            if (isSolanaEnabled()) {
-                keyRings.add(BraveWalletConstants.SOLANA_KEYRING_ID);
-            }
-            if (isFilecoinEnabled()) {
-                keyRings.add(BraveWalletConstants.FILECOIN_KEYRING_ID);
-                keyRings.add(BraveWalletConstants.FILECOIN_TESTNET_KEYRING_ID);
-            }
-            return keyRings.toArray(new String[0]);
-        }
-
```

Comment:
Backend should be the only source of truth for this and already has similar method.